hack/verify-flags/known-flags.txt

```diff
@@ -12,6 +12,7 @@ allowed-not-ready-nodes
 anonymous-auth
 api-advertise-addresses
 api-external-dns-names
+api-port
```

Comment:
It doesn't seem like this file is sorted properly... is it supposed to be? Also, I don't seem to find some of the kubeadm flags in there, how come?